app/config.js

```diff
@@ -43,7 +43,10 @@ config.auth0 = {
 };
 
 config.aws = {
-  login_url: process.env.AWS_LOGIN_URL,
+  login_url: {
+    prefix: `https://aws.services.${process.env.ENV}.mojanalytics.xyz/?destination=/s3/buckets/`,
+    suffix: '/?region=eu-west-1&tab=overview',
```

Comment:
Shouldn't the `destination` value be `escape`d? and the `region` and `tab` doubly escaped?

Reply:
Also, it would be better not to hard-code this base URL - the AWS login proxy thing might be deployed somewhere else (like on some `digital.justice.gov.uk` subdomain, for a likely example).
Better to split this up, like:
```javascript
config.aws = {
  login_base_url: process.env.AWS_LOGIN_BASE_URL || `https://aws.services.${process.env.ENV}.mojanalytics.xyz`,
  bucket_url: (name, region = 'eu-west-1') => {
    return `${config.aws.login_base_url}/?destination={escape(`/s3/buckets/${name}/?region=${region}&tab=overview`)}`,
  },
};
```
(might not be the best place for this function, but you get the idea